tests/test_spec.py

```
@@ -1443,6 +1443,53 @@ async def test_revise_spec_spawn_failure(self) -> None:
 
         mock_clear.assert_not_called()
 
+    async def test_approve_transitions_issue_to_researched(self, _spec_dir: Path) -> None:
```

Ruff (0.16.1)

[warning] Fixture `_spec_dir` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)